src/daemons/rhsmcertd.c

```diff
@@ -474,11 +450,11 @@ static gboolean
 initial_cert_check (gpointer data)
 {
     struct CertCheckData *cert_data = data;
-    cert_check (cert_data->heal);
+    cert_check (data);
```

Comment:
simply pass `NULL` here

Reply:
While I was on vacation it dawned on me that this change would be needed. I'm glad you pointed it out too.
Addressed by commit: [84272c3](https://github.com/candlepin/subscription-manager/pull/3455/commits/84272c302b6063aeb18b95b6150b068293489d81)